LondonBritishLibrary/orient/BLorient12859.xml

```diff
@@ -0,0 +1,437 @@
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <title ref="LIT2866RepCh147#10" type="incomplete">Salām to George of Lydda</title>
+                            <note>Stanza 10 of <ref type="work" corresp="LIT2866RepCh147"/>, written in the left and middle columns under the 
+                                fifth picture.</note>
+                            <incipit xml:lang="gez">ሰላም፡ ለከናፍሪከ፡ እለ፡ ተከሥታ፡ ለዘምሮ። ስመ፡ ኢየሱስ፡ ክርስቶስ፡ መምለክያነ፡ ጣዖት፡ ለአስተሐፍሮ። ጊዮርጊስ፡ ልዳዊ፡ ዕጉሥ፡ 
+                                ለተጽእሮ። ፈረስከ፡ ዘትፄዓኖ፡ እንደ፡ ያስተቃንጽ፡ ዕግሮ። ለረዊደ፡ ጸርየ፡ ይሩጽ፡ ወኢያንትግ፡ በጽሮ።</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <title ref="LIT7120Respect"/>
+                            <note>Written in the right column under the fifth picture and in the left and middle column under the sixth picture.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ዑን፡ ነቢ፡ ሰላማ፡ <gap reason="ellipsis"/> ከማሁ፡ አሸብሮሙ፡ ለሰብአ፡ ኵሉ፡ ፍጥረት፡
+                                በግርማ፡ ገጸ፡ ገብርከ፡ <gap reason="lost"/></incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i7">
+                            <title ref="LIT5888AsmatPrayer">Prayer for gaining respect from Gāllā, Šānqǝllā and all kinds of people</title>
+                            <note>Written in the right column under the sixth picture and in the three columns under the seventh picture. The text is 
+                                partly similar to the prayer above, but using several 
+                                <term key="Magic">magic words (<foreign xml:lang="gez">ʾasmāt</foreign>)</term> at the beginning. 
+                                The letter <foreign xml:lang="gez">ዥ</foreign> is represented by the more ancient or irregular form with four little 
+                                strokes marking the sixth order: <figure><graphic url="https://betamasaheft.eu/resources/images/snippets/BLorient12996/sign2.png" height="30px">
+                                <desc>Symbol as represented in the catalogue by Stefan Strelcyn</desc></graphic></figure>.</note>
+                            <incipit xml:lang="gez">ዥኖ፡ ከሚጽ፡ ከደቢል፡ እልሚል፡ <gap reason="ellipsis"/> ከማሁ፡ አሸብሮሙ፡ ለሰብአ፡ ጋላ፡ ወለሰብአ፡ ሻንቅላ፡ ወለሰብአ፡
+                                ኵሉ፡ ፍጥረት፡ በግርማ፡ ገጸ፡ ገብርከ፡ <gap reason="lost"/></incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i8">
+                            <title ref="LIT5888AsmatPrayer">Prayer for gaining respect from Gāllā, Šānqǝllā and all kinds people</title>
```

Comment:
all kinds of people?

Reply:
This is how I understand ለሰብአ፡ ጋላ፡ ወለሻንቅላ፡ ወለሰብአ፡ ኵሉ፡ ፍጥረት፡ 

I am open for other interpretations.

Reply:
A more literal translation might be "and people of all of creation"/"people of the whole creation"

Reply:
Now, I understand Eugenias request: "of" was missing in the text.

I think, that "all kinds of people" is better in this context.